packages/web/pages/apps/index.tsx

```diff
@@ -80,6 +84,25 @@ export const AppStore: React.FC<AppStoreProps> = ({ apps }) => {
 
   const iterableData = searchValue ? fuzzySearchResults : nonFeaturedApps;
 
+  const getSize = useCallback(() => {
+    if (width <= 480) {
+      return "small";
+    } else if (width <= 900) {
+      return "medium";
+    } else if (width <= 1500) {
+      return "long";
+    }
+    return "large";
+  }, [width]);
+
+  const [searchBoxSize, setSearchBoxSize] = useState<
```

Comment:
This is not a functional use of state and useEffect. Not idiomatic React. 

We should simply have the size be a function of the current width for each render. (What's in the useCallback).